src/Tabs/index.jsx

```diff
@@ -43,25 +44,20 @@ class Tabs extends React.Component {
       const labelId = this.genLabelId(i);
 
       return (
-        <li
-          className={styles['nav-item']}
+        <Button
+          role="tab"
+          aria-selected={selected}
+          aria-controls={this.genPanelId(i)}
           id={labelId}
           key={labelId}
-        >
-          <a
-            aria-selected={selected}
-            aria-controls={this.genPanelId(i)}
-            className={classNames(
-              styles['nav-link'],
-              { [styles.active]: selected },
-            )}
-            onClick={() => { this.toggle(i); }}
-            role="tab"
-            tabIndex={selected ? 0 : -1}
-          >
-            {label}
-          </a>
-        </li>
+          onClick={() => { this.toggle(i); }}
+          className={classNames(
+            styles['nav-link'],
+            styles['nav-item'],
+            { [styles.active]: selected },
+          ).split(' ')}
```

Comment:
I think if we're planning on making a breaking change to make Buttons accept strings instead of arrays (like they should), I like that this is an obvious work around. An array of 1 string with spaces in it is less likely to be caught in the refactor.